policy.go

```diff
@@ -485,6 +492,61 @@ func computeStaticPolicy(alg tpm2.HashAlgorithmId, input *staticPolicyComputePar
 		PinIndexAuthPolicies: input.pinIndexAuthPolicies}, trial.GetDigest(), nil
 }
 
+// computePolicyORData computes data required to perform a sequence of TPM2_PolicyOR assertions in order to support compound
+// authorization policies with more than 8 conditions (which is the limit of the TPM). Its main purpose is to support PCR policies
+// with more than 8 conditions. It works by turning a list of digests (or, conditions) in to a tree of nodes, with each node
+// containing no more than 8 digests that can be used in a single TPM2_PolicyOR assertion, the root of the tree containing digests
+// for the final TPM2_PolicyOR assertion, and leaf nodes containing digests for each OR condition. Whilst the returned data is
+// conceptually a tree, the layout in memory is just a slice of tables of up to 8 digests, each with an index that enables the code
+// executing the assertions to traverse upwards through the tree by just advancing to another entry in the slice. This format is
+// easily serialized. After the computations are completed, the provided *tpm2.TrialAuthPolicy will be updated.
+//
+// The returned data is used by firstly finding the leaf node which contains the current session digest. Once this is found, a
+// TPM2_PolicyOR assertion is executed on the digests in that node, and then the tree is traversed upwards to the root node, executing
+// TPM2_PolicyOR assertions along the way - see executePolicyORAssertions.
+func computePolicyORData(alg tpm2.HashAlgorithmId, trial *tpm2.TrialAuthPolicy, digests tpm2.DigestList) policyOrDataTree {
+	var current int
```

Comment:
current is an index in data, it should be called curNode, also for clary I would define it after the tree and list, as:

curNode := 0